keyboards/kradoindustries/promenade_rp24s/keyboard.json

```diff
@@ -0,0 +1,140 @@
+{
+    "keyboard_name": "Promenade RP24S",
+    "manufacturer": "Krado Industries",
+    "url": "https://www.kradoindustries.com/",
+    "maintainer": "Krado Industries",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x4B72",
+        "pid": "0x4B6D",
+        "device_version": "1.0.0"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "encoder": true,
+        "rgblight": true
+    },
+
+    "encoder": {
+        "rotary": [
+            { "pin_a": "GP12", "pin_b": "GP15" },
+            { "pin_a": "GP12", "pin_b": "GP16" },
+            { "pin_a": "GP15", "pin_b": "GP18" },
+            { "pin_a": "GP16", "pin_b": "GP18" },
+            { "pin_a": "GP11", "pin_b": "GP15" },
+            { "pin_a": "GP16", "pin_b": "GP11" },
+            { "pin_a": "GP16", "pin_b": "GP15" },
+            { "pin_a": "GP18", "pin_b": "GP12" }
+        ]
+    },
+
+    "ws2812": {
+        "pin": "GP26",
+        "driver": "vendor"
+    },
+    "rgblight": {
+        "led_count": 14,
+        "sleep": true,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
```

Comment:
```suggestion
```

Reply:
Why is it necessary to remove rgb_test from animations?

Reply:
Generally, it's not explicitly required.  But from a user standpoint, this animation doesn't make sense to have enabled, but it does make sense for a vendor to have access to, as it will confirm that all of the LEDs are working properly.

Reply:
Thank you for the explanation. I would prefer to leave it in, as there is no shortage of memory on the board.